setup.cfg

```diff
@@ -0,0 +1,17 @@
+[metadata]
+name = python_stub
+author = Daniel Huppmann
+author_email = huppmann@iiasa.ac.at
+description = Template for creating python packages and Sphinx-based docs pages
```

Comment:
Could add:
- `long_description`, e.g. https://github.com/iiasa/ixmp/blob/5c4a78c7198e73d0b68414054c33ac1a16925f09/setup.cfg#L7-L8
- A commented line like `# license = (TODO: update this to match the license you choose for the package)`

Reply:
Nice suggestions, done